Source/WebCore/inspector/InspectorCanvas.cpp

```diff
@@ -113,40 +113,40 @@ InspectorCanvas::InspectorCanvas(CanvasRenderingContext& context)
 
 HTMLCanvasElement* InspectorCanvas::canvasElement() const
 {
-    return dynamicDowncast<HTMLCanvasElement>(m_context.canvasBase());
+    return dynamicDowncast<HTMLCanvasElement>(m_context->canvasBase());
 }
 
 ScriptExecutionContext* InspectorCanvas::scriptExecutionContext() const
 {
-    return m_context.canvasBase().scriptExecutionContext();
+    return m_context->canvasBase().scriptExecutionContext();
 }
 
 JSC::JSValue InspectorCanvas::resolveContext(JSC::JSGlobalObject* exec) const
 {
     JSC::JSLockHolder lock(exec);
-
+    auto& context = const_cast<CanvasRenderingContext&>(m_context.get());
```

Comment:
Have you tried updating the InspectorCanvas::resolveContext() function to not be `const` instead of doing a const_cast here?

Reply:
You're right, that would work, I'll fix this